lib/components/artist/__tests__/about-tests.js

```diff
@@ -0,0 +1,19 @@
+import 'react-native'
+
+import React from 'react'
+import renderer from 'react-test-renderer'
+
+import About from '../about'
+
+jest.mock('../../opaque_image_view.js', () => 'AROpaqueImageView')
+
+it('renders properly', () => {
+  const artist = {
+    has_metadata: true,
+    is_display_auction_link: true,
+    articles: [],
```

Comment:
so, we're testing and rendering each component separately? So no articles here (within 'about'), but instead in its own test, as per below? (`lib/components/artist/__tests__/articles-tests.js`)